roles/api-environment/tasks/main.yml

```diff
@@ -67,8 +67,8 @@
 
   - name: "setup environment for dropwizard api"
     import_tasks: dropwizard-api.yml
-    when: dropwizard_api
+    when: dropwizard_api|bool
 
   - name: "setup environment for express api"
     import_tasks: express-api.yml
-    when: express_api
+    when: express_api|bool
```

Comment:
From my [comment](https://github.sig.oregonstate.edu/ecs-data/ansible-private-roles/pull/355#discussion_r914) on the private roles repo, these two when statements shouldn't need the `bool` filter since the values should be booleans